src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,45 @@
+package calculator;
+
+public class StringCalculator {
+    static void validateIsNull(String input) {
+        if (input == null || input.trim().isEmpty()) {
+            throw new IllegalArgumentException("Input is Null");
+        }
+    }
+
+    static void validateCalc(String calc) {
+        if (!"+".equals(calc) && !"-".equals(calc) && !"*".equals(calc) && !"/".equals(calc)) {
+            throw new IllegalArgumentException("This is not Calculate");
+        }
+    }
+
+    static int operate(int a,int b,String calc) {
```

Comment:
```suggestion
    static int operate(int a, int b, String calc) {
```
전체적으로 메소드의 파라미터들의 정렬이 조금 맞지 않는 것 같아요! 위에 남겨둔 내용을 참고해주세요